packages/lib-utils/src/components/list-view/ListView.tsx

```diff
@@ -53,78 +72,83 @@ const ListView: React.FC<ListViewProps<Record<string, unknown>>> = ({
+        inputValue.current?.length > 0
+          ? { ...filterValues.current, [activeFilter.id]: [inputValue.current] }
+          : omit(filterValues.current, activeFilter.id),
+      );
+    }
+
+    setUseUrl(true);
+  }, 2000);
 
   return (
     <>
       <Toolbar>
         <ToolbarContent>
           {filters ? (
             <>
-              <ToolbarItem key="filter-select">
-                <Select
-                  toggleIcon={<FilterIcon />}
-                  variant={SelectVariant.single}
-                  onToggle={(value) => setFilterSelectExpanded(value)}
-                  onSelect={(e, selection) => {
-                    setActiveFilter(filters.find((item) => item.id === selection));
-                    setFilterSelectExpanded(false);
-                  }}
-                  placeholderText={activeFilter?.label}
-                  isOpen={isFilterSelectExpanded}
-                >
-                  {filters.map((option) => (
-                    <SelectOption key={option.id} value={option.id}>
-                      {option.label}
-                    </SelectOption>
-                  ))}
-                </Select>
-              </ToolbarItem>
+              {filters.length > 1 && (
+                <ToolbarItem key="filter-select">
+                  <Select
+                    toggleIcon={<FilterIcon />}
+                    variant={SelectVariant.single}
+                    onToggle={(value) => setFilterSelectExpanded(value)}
+                    onSelect={(e, selection) => {
+                      setActiveFilter(filters.find((item) => item.id === selection));
+                      setFilterSelectExpanded(false);
+                    }}
+                    placeholderText={activeFilter?.label}
+                    isOpen={isFilterSelectExpanded}
+                  >
+                    {filters.map((option) => (
+                      <SelectOption key={option.id} value={option.id}>
+                        {option.label}
+                      </SelectOption>
+                    ))}
+                  </Select>
+                </ToolbarItem>
+              )}
               <ToolbarItem variant={ToolbarItemVariant['search-filter']} key="search-filter">
                 <SearchInput
                   className="dps-list-view__search"
                   onChange={(value) => {
-                    if (activeFilter) {
-                      const newValues =
-                        value?.length > 0
-                          ? { ...filterValues.current, [activeFilter.id]: [value] }
-                          : omit(filterValues.current, activeFilter.id);
-                      setFiltersToURL(
-                        searchParams,
-                        setSearchParams,
-                        filters.map((filter) => filter.id),
-                        newValues,
-                      );
-                    }
+                    if (useUrl) setUseUrl(false);
```

Comment:
Normally this would trigger an ESLint warning saying you should encapsulate `if` body inside `{` `}` braces to make it more readable - if it doesn't, maybe we should enforce this rule?

Reply:
@vojtechszocs added the curly braces, just not sure the eslint rule. Added it to the config but it seems not to work. Could you please take a look? Thank you!